admin/git.md

```diff
@@ -122,6 +137,23 @@ When done, click **Save**.
 >       value: ""
 > ```
 
+#### Version 7.20 or later
+
+On your Bitbucket Server, go to **Administration** > **Applications** >
+**Application Links** and select _Create link_.
+
+- Set **Application type** to _External application_
+- Set **Direction** to _Incoming_
+
+Click _Continue_.
+
+- Enter a unique name for the link, e.g. "Coder"
+- Set **Redirect URL** to `[your-coder-domain]/oauth/callback` (e.g.
+  `https://coder.domain.com/oauth/callback`)
+- In **Application permissions**, under **Account**, enable _Write_ permission
+
+Click _Save_ and enter the generated **Client ID** and **Client Secret**.
+
```

Comment:
suggestion: Assuming that folks are more likely to use a newer version of Bitbucket, should we move these instructions to the top, and move the instructions for <= 7.19 below?

Reply:
That sounds sensible